stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketUtil.java

```diff
@@ -26,27 +26,66 @@
 import org.ballerinalang.jvm.BallerinaValues;
+import org.ballerinalang.jvm.scheduling.Strand;
 import org.ballerinalang.jvm.types.BPackage;
+import org.ballerinalang.jvm.types.BType;
 import org.ballerinalang.jvm.values.ErrorValue;
 import org.ballerinalang.jvm.values.MapValue;
 import org.ballerinalang.jvm.values.ObjectValue;
 import org.ballerinalang.jvm.values.connector.NonBlockingCallback;
 import org.ballerinalang.net.http.HttpConstants;
 import org.ballerinalang.net.http.HttpErrorType;
+import org.ballerinalang.net.http.HttpUtil;
+import org.ballerinalang.net.http.websocket.client.FailoverContext;
+import org.ballerinalang.net.http.websocket.client.RetryContext;
+import org.ballerinalang.net.http.websocket.client.WebSocketClientHandshakeListener;
+import org.ballerinalang.net.http.websocket.client.WebSocketClientListener;
+import org.ballerinalang.net.http.websocket.client.WebSocketFailoverClientHandshakeListener;
+import org.ballerinalang.net.http.websocket.client.WebSocketFailoverClientListener;
 import org.ballerinalang.net.http.websocket.server.WebSocketConnectionManager;
 import org.ballerinalang.net.http.websocket.server.WebSocketOpenConnectionInfo;
 import org.ballerinalang.net.http.websocket.server.WebSocketServerService;
+import org.ballerinalang.net.http.websocket.server.WebSocketService;
 import org.ballerinalang.stdlib.io.utils.IOConstants;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.wso2.transport.http.netty.contract.HttpWsConnectorFactory;
+import org.wso2.transport.http.netty.contract.websocket.ClientHandshakeFuture;
+import org.wso2.transport.http.netty.contract.websocket.WebSocketClientConnector;
+import org.wso2.transport.http.netty.contract.websocket.WebSocketClientConnectorConfig;
+import org.wso2.transport.http.netty.contract.websocket.WebSocketCloseMessage;
 import org.wso2.transport.http.netty.contract.websocket.WebSocketConnection;
 
 import java.io.IOException;
+import java.net.URI;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
 import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 import javax.net.ssl.SSLException;
 
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.BACK_OF_FACTOR;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.CLIENT_CONNECTOR;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.CLIENT_LISTENER;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.CONNECTOR_FACTORY;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.COUNT_DOWN_LATCH;
 import static org.ballerinalang.net.http.websocket.WebSocketConstants.ErrorCode;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.FAILOVER_CONFIG;
+import static org.ballerinalang.net.http.websocket.WebSocketConstants.FAILOVER_INTEVAL;
```

Comment:
Shall we please not use static imports in all places unless it makes sense: https://stackoverflow.com/a/162195/3599535
It confuses future readers of the code. I personally found it difficult to read the code where you had used static imports.